src/issues/issueFeatureRegistrar.ts

```diff
@@ -481,6 +482,15 @@ ${body ?? ''}\n
 		editor.setDecorations(labelsDecoration, [new vscode.Range(new vscode.Position(3, 0), new vscode.Position(3, labelLine.length))]);
 	}
 
+	private async verifyLabels(createParams: Octokit.IssuesCreateParams) {
+		if (!createParams.labels) {
+			return;
+		}
+		const allLabels = await this.manager.getLabels(undefined, createParams);
+		const filteredLabels = allLabels.filter(label => createParams.labels?.includes(label.name)).map(label => label.name);

```

Comment:
I think it would be good to give some kind of feedback when a label isn't found. Maybe we could show a confirmation dialog showing the label with options "Proceed anyway", "Create label", and "Cancel"?

Reply:
If we do decide to do that, I think it would also be good to verify assignees and similarly show a dialog if validation fails